packages/logs-ppx/logs-ppx.0.1.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+version: "0.1.0"
+synopsis: "PPX to cut down on boilerplate when using Logs"
+description:
+  "PPX to remove the closure when logging with Logs. The PPX will still generate the closure which keeps the nice properties of the Logs library."
+maintainer: ["ulrik.strid@outlook.com"]
+authors: ["Ulrik Strid"]
+license: "BSD-3"
+homepage: "https://github.com/ulrikstrid/logs-ppx"
+bug-reports: "https://github.com/ulrikstrid/logs-ppx/issues"
+depends: [
+  "ocaml" {>= "4.8.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.08.0"}
```

Reply:
Just a side note: the two syntaxes are equivalent

Reply:
Yes, that is why I did not say anything. It was mostly to keep it consistent in the repo